blocks/editable/provider.js

```diff
@@ -6,7 +6,7 @@ import { pick, noop } from 'lodash';
 /**
  * WordPress dependencies
  */
-import { Component } from '@wordpress/element';
+import { Component, Children } from '@wordpress/element';
```

Comment:
Can we avoid the `Children` utility and instead leverage the fact that `children` is an array in Preact, i.e. 

```js
// Before:
Children.only( this.props.children );

// After:
this.props.children[ 0 ];
```

I think it would be good if we avoid `Children` altogether.

Reply:
That's the exact same thing, the inner implementation is the same, but I can see that this enables to drop `preact-compat` at some moment.